src/source-af-packet.c

```diff
@@ -736,6 +738,18 @@ static inline int AFPSuriFailure(AFPThreadVars *ptv, union thdr h)
     SCReturnInt(AFP_SURI_FAILURE);
 }
 
+void AFPReadCopyBypass(Packet *dst, Packet *src)
+{
+#ifdef HAVE_PACKET_EBPF
+    dst->BypassPacketsFlow = src->BypassPacketsFlow;
```

Comment:
Why does the `Packet` structure hold this information ? (and not the structure related to the interface)

This way, tunneled packets did not have `BypassPacketsFlow` set and `AFPXDPBypassCallback` did not even need to check `PacketIsTunnel`